### DISCUSSION TITLE
I am new to babylonJs and Typescript . Can you please help  me adding the morph Targets Below is my code.

### DISCUSSION THREAD

**Discussion by steHawk** [brianzinn/react-babylonjs#239] [Q&A]
<Model
                  rootUrl={`https://raw.githubusercontent.com/ml-bhanuprakash/babylonjs/main/test_2/`}
                  sceneFilename={`Test_lady.glb`}
                  scaleToDimension={0}
                  // morphTargetManager={ MorphTargetManager. }
                  // scaling={new Vector3(1,1,1)}
                  // position={new Vector3(1, -1, 1)}

                  // onModelLoaded={(): void => onModelLoaded(model)}
                  name={""}
                />

**Comment by dennemark** [comment]
Looking at this: https://doc.babylonjs.com/divingDeeper/mesh/morphTargets
You could try getting the MorphTargetManager somehow like this:
```

const [meshInfluences, setMeshInfluences] = useState()
return <Model
              rootUrl={`https://raw.githubusercontent.com/ml-bhanuprakash/babylonjs/main/test_2/`}
              sceneFilename={`Test_lady.glb`}
              scaleToDimension={0}
              onModelLoaded={(m) => {
              let influences = []
              m.meshes.forEach(mesh => {
                 mesh.material = loaderMat;
                   if (mesh.morphTargetManager != null) {
                     influences.push(mesh.morphTargetManager.getTarget(0));
                    }
                  });
                 setMeshInfluences(influences);
                }}
              name={""}
            />
```
Then you can use meshInfluences in i.e. useEffect or somewhere else. But it wont be declarative react, but imperative babylonjs code, so then you can follow the babylon docs.
I havent used morph targets myself though.
